Assets/Scripts/MapGen/MapGeneration.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Tilemaps;
+
+public class MapGeneration : MonoBehaviour
+{
+
+    public Tilemap tilemap;
+    
+    public TileBase ground1;
+    public TileBase ground2;
+    public TileBase air;
+
```

Comment:
Please use capital on public variables, also is it necessary to make them public?

Reply:
fixed